requirements.txt

```diff
@@ -12,3 +12,4 @@ xvfbwrapper
 psutil
 funcsigs
 configparser
+nilearn
```

Comment:
haha! not in requirements.txt as far as i am concerned.  we don't depend on fsl, spm, nipy, matplotlib, etc.,.

so these tests should all have things like `@skip_if(no_nilearn)` but it does seem that masking is so trivial we could achieve it with nibabel alone.